client/block.c

```diff
@@ -482,6 +522,10 @@ static int add_block_nolock(struct xdag_block *newBlock, xdag_time_t limit)
 		}
 	}
 
+#if CACHE == 1
```

Comment:
What is the purpose of caching?

Reply:
To avoid this bottleneck:
if (newBlock->field[i].amount) {
					struct xdag_block buf;
	this	----------->	        struct xdag_block *bref = xdag_storage_load(blockRef->hash, blockRef->time, blockRef->storage_pos, &buf);
					if (!bref) {
						err = 8;
						goto end;
					}

Reply:
cache hitrate is around 90/95% so it seems that works good to avoid it